internal/cmd/pipeline/command_create.go

```diff
@@ -68,14 +68,24 @@ func (c *command) create(cmd *cobra.Command, _ []string) error {
 	}
 
 	// validate ksql id
-	if _, err := c.V2Client.DescribeKsqlCluster(ksqlCluster, environmentId); err != nil {
-		return err
+	if ksqlCluster != "" {
+		if _, err := c.V2Client.DescribeKsqlCluster(ksqlCluster, environmentId); err != nil {
+			return err
+		}
 	}
 
 	// validate sr id
+	srId := ""
 	srCluster, err := c.Config.Context().SchemaRegistryCluster(cmd)
 	if err != nil {
-		return err
+		if !strings.Contains(err.Error(), "Schema Registry not enabled") {
```

Comment:
if sr is not enabled, it will continue to create?

Reply:
Yes, thats the new feature.